src/lib/utils/index.ts

```diff
@@ -140,3 +140,45 @@ export async function createGitRepository() {
   await git.raw(['add', '-A'])
   await git.raw(['commit', '-m', 'initial commit'])
 }
+
+export function requireModule(config: {
+  depName: string
+  optional: boolean
+}): null | unknown {
+  const depPath = process.env.LINK
+    ? Path.join(process.cwd(), '/node_modules/', config.depName)
+    : config.depName
+
+  try {
+    const dep = require(depPath)
+    // The code may have been compiled from a TS source and then may have a .default property
+    if (dep.default !== undefined) {
```

Comment:
We could use the `tslib`'s `__importDefault` instead? https://github.com/microsoft/tslib/blob/master/tslib.js#L235 🤔

Reply:
Yeah looks like we can do something a lot better here. But not exactly sure what just yet.